app/build.gradle

```diff
@@ -40,4 +39,9 @@ dependencies {
     compile 'com.android.support:support-v4:24.2.1'
     compile 'com.squareup.retrofit2:retrofit:2.1.0'
     compile 'com.squareup.retrofit2:converter-gson:2.1.0'
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.mockito:mockito-core:1.9.5'
```

Comment:
nit-picking: There's no need to shuffle this around. I do see though that you added Picasso but shuffling it around creates a difference b/n this and the original file and distracts your reviewer :)